src/hypothesis/searchstrategy/flatmapped.py

```diff
@@ -42,3 +42,11 @@ def do_draw(self, data):
         source = data.draw(self.flatmapped_strategy)
         data.mark_bind()
         return data.draw(self.expand(source))
+
+    @property
+    def branches(self):
+        branches = [
```

Comment:
Only that this was originally a bit more complicated in the early versions and I couldn’t do a one-line return. Will fix.